zulipterminal/cli/run.py

```diff
@@ -388,7 +388,11 @@ def main(options: Optional[List[str]]=None) -> None:
         if args.color_depth:
             zterm['color-depth'] = (args.color_depth, 'on command line')
 
-        color_depth = int(zterm['color-depth'][0])
+        color_depth_str = zterm['color-depth'][0]
+        if color_depth_str == '2**24':
+            color_depth = 2 ** 24
```

Comment:
This would be better as `24bit`; we can migrate others across to support similar naming too, perhaps.